frontend/.eslintrc.json

```diff
@@ -60,25 +64,24 @@
       "files": ["*.ts"],
       "extends": [
         "airbnb-typescript/base",
-        "plugin:@typescript-eslint/recommended"
+        "plugin:@typescript-eslint/recommended",
+        "plugin:@stylistic/disable-legacy"
```

Comment:
`disable-legacy` had to be re-added here to disable `typescript-eslint/recommended` old linting rules from interfering with the new linting rules.